script/1_0_2current_en_script/Arcueid/Day 5/05_00_ARC05_6_2.txt

```diff
@@ -39,145 +39,145 @@ That face... Is she enjoying cornering him...?
 She is inviting Vlov to perform one final attack. A counterattack with all his might.
 <Page16>
 Countless clashing sounds echo through the air.
 Vlov's breathing grows even more frantic.
 "―――No, that's―――"
-A sense of uneasiness that I cannot put into words.
-Arcueid has the upper hand. It's clear that Vlov is getting exhausted.
+A sense of unease that I can't put into words.
+Arcueid has the upper hand. It's clear that Vlov is being worn down.
 And yet as a bystander, I feel a growing sense of dread in my heart.
 <Page17>
-The source of the dread is Vlov's increasing invulnerability to Arcueid's claws.
+The dread stems from the fact that Vlov is taking far fewer hits from Arcueid's claws than he was a little while ago.//それは、ヴローヴが先ほどからアルクェイドの爪を受けなくなりつつある状況であり、
 <Page18>
-It's also his swordsmanship... No, his skill with that billhook is much greater than one would expect from his appearance.
+His swordsmanship... or rather, machetemanship, is also much more formidable than one would expect given his appearance.//Made up word but kinda matches the joke better ヤツの剣さばき……いや、鉈さばきは...
 <Page19>
-Furthermore, it's also the fact that he's already able to completely see through Arcueid's speed and movements. And most important of all―――
+On top of that, it's clear that he's already able to see right through Arcueid's speed and movements. But most importantly of all―――
 He still hasn't used that 'flaming hand' of his...!
 <Page20>
 "GU――――――AAAAAAAAAAH!!!!!"
 "―――!"
 <Page21>
 Vlov lets out a roar.
-The flames blaze in response to his mighty roar of rage.
-The eternal flames reach a height of several meters,
-dividing the river into dozens of branches.
+The flames blaze in response to his bellow of rage.
+The apocalyptic flames reach a height of several meters,//Not sure what a good TL is for 劫火
+and the river of fire splits into dozens of streams.
 <Page22>
 The underground arena turns into a forest of fire.
-The wall of flames prevents Arcueid's escape.
-And from there,
+The wall of flames blocks off Arcueid's escape.
+And with the stage set―――
 <Page23>
-a hand of unparalleled heat is unleashed towards Arcueid, melting the rubble in its path.
+A 'hand' of unparalleled heat is unleashed towards Arcueid, melting the rubble in its path.
 <Page24>
 "Arcueid―――!"
-In one instant.
-The tides of battle are turned in an instant.
+In an instant.
+The tide of battle is completely reversed.//Second 一瞬 elided, felt redundant 戦いは一瞬でひっくり返った
 <Page25>
 Arcueid said that she could withstand temperatures of up to 3000℃.
 However, the 'hand' that melted the rubble with ease had a temperature well over that.
 <Page26>
-He could even stack three of them, in the face of which not even ashes can remain. The flames become a tormenting tongue, playing with their prey who are unable to escape.
+Three more of those 'hands' are piled on. Not even ashes will remain at this rate. There's no escape from the tormenting tongue of flame as it licks its prey clean.
```

Comment:
I guess we never did get around to fixing this after group PR so thanks for the change.